### ISSUE TITLE
Problem with email mask on Android devices

### ISSUE THREAD

**Issue by tlonovoi** [RobinHerbots/jquery.inputmask#2556]
When you start to type, plugin adds a character and jumps to the next part of email string after @ sign
Also strange behavior when trying to delete symbols with Android backspace key

**Comment by sheck87** [comment]
Yes, on mobile phones there are some issues with email editing. Here video example: https://disk.yandex.ru/i/rZqUvH17zDSHoA

**Comment by chris-wilson-cs** [comment]
We are also seeing this issue and have tried upgrading to version [5.0.9](https://github.com/RobinHerbots/Inputmask/releases/tag/5.0.9) and the issue is still present there.
